tests/spec/fs.chmod.spec.js

```diff
@@ -39,6 +39,16 @@ describe('fs.chmod, fs.fchmod', function() {
     });
   });
 
+  it('should be an error when the path is invalid', function(done){
```

Comment:
I think the wording here should be more about the method _doing_ something rather than _being_ something. Perhaps something like "it should throw an error" would be better?

Having taken a look at existing tests, there is a lot of variation on how people have chosen to title the tests. I've seen both "it should fail" and "it should return an error" in various places.